### PR TITLE
[stable/suitecrm] Add ingress

### PR DESCRIPTION
**Pull request by chriskolenko**
Using other helm charts as a reference I've added in an Ingress template.
- enabled pattern with default to false
- standard annotations + tls values

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/suitecrm/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: suitecrm
-version: 5.0.1
+version: 5.0.2
```
Comment (davidkarlsen) [inline stable/suitecrm/Chart.yaml]: new feature
```suggestion
version: 5.1.0
```

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

**Comment by chriskolenko** [comment]
@tompizmor or @carrodher any chance for a review? I've tested this change in a dev cluster with nginx-ingress + cert-manager + external-dns successfully.

**Comment by chriskolenko** [comment]
> Thanks for the contribution! Changes LGTM, but I would add more information to the NOTES.txt. I mean, some explanation in the ingress `if` like we are doing in the `NodePort` or `ClusterIP`, what do you think?

Damn, it's like you knew where I was struggling. I only quickly patched the NOTES.txt because of the number of if statements. My brain got a little lost.

After reviewing what I wrote. I'm believe it could be made better if we treated Ingress as an unconditional extra instead of an if/else. I.E. If ingress is enabled.. you get a list of the ingress host names and all the Service information. 

Let me change it quickly. Sorry about this message, it came to me writing this comment.

**Comment by tompizmor** [comment]
Hello @chriskolenko,

Thanks for sending this PR. I am not sure which other charts you have used as a reference but I miss a couple of small features. Please check this other recent PR as reference https://github.com/helm/charts/pull/10335

Precisely, I am missing to be able to provide [custom tls certificates](https://github.com/helm/charts/blob/master/stable/prestashop/values.yaml#L195-L207) from the values.yaml and the [certManager switch](https://github.com/helm/charts/blob/master/stable/prestashop/values.yaml#L171-L172).

Apart from that, please rebase from master to fix te conflicts.

**Pull request closed without merging**